README.md

```diff
@@ -136,28 +132,19 @@ DISCOVERY_COLLECTION_ID=<add_discovery_collection>
 
 ![](doc/source/images/sample-output.png)
 
-# Links
-
-* [Demo on Youtube](https://youtu.be/zAu9tHefdDc)
-* [Blog: Bring your own data to Watson Discovery Service](doc/index.md)
-* [Watson Node.js SDK](https://github.com/watson-developer-cloud/node-sdk)
-
 # Troubleshooting
 
 * Error: Environment {GUID} is still not active, retry once status is active
 
-  > This is common during the first run. The app tries to start before the Discovery environment is fully created. Allow a minute or two to pass. The environment should be usable on restart. If you used `Deploy to Bluemix` the restart should be automatic.
+  > This is common during the first run. The app tries to start before the Discovery environment is fully created. Allow a minute or two to pass. The environment should be usable on restart. If you used `Deploy to IBM Cloud` button the restart should be automatic.
 
 * Error: Only one free environent is allowed per organization
 
   > To work with a free trial, a small free Discovery environment is created. If you already have a Discovery environment, this will fail. If you are not using Discovery, check for an old service thay you may want to delete. Otherwise use the .env DISCOVERY_ENVIRONMENT_ID to tell the app which environment you want it to use. A collection will be created in this environment using the default configuration.
 
-# License
-[Apache 2.0](LICENSE)
-
 # Privacy Notice
-If using the `Deploy to Bluemix` button some metrics are tracked, the following
-information is sent to a [Deployment Tracker](https://github.com/IBM/metrics-tracker-service) service
+
+If using the Deploy to IBM Cloud button some metrics are tracked, the following information is sent to a [Deployment Tracker](https://github.com/IBM-Bluemix/cf-deployment-tracker-service) service
```

Comment:
missed the back ticks around `Deploy to IBM Cloud`

Reply:
This change looks like it is going from the new tracker to the one.  Not sure what's going on there.